src/pages/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/troubleshooting.md

```diff
@@ -62,3 +70,43 @@ If you are using the default NFS storage however, then the script pod would be d
 
 - being evicted due to exceeding its storage quota
 - being moved or restarted as part of routine cluster operation
+
+## Frequently Asked Questions {#FAQ}
+
+### Can the agent work with Octopus running in an HA Cluster setup?
+Yes! See the [Kubernetes agent HA Cluster Support](/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/ha-cluster-support) page.
+
+
+### Can a proxy be specified when setting up the Kubernetes agent? 
+Yes! Proxy servers for the polling connection that takes place between the agent and Octopus Server. These can be supplied for setup via the `.pollingProxy.*`  helm values.
```

Comment:
the second sentence feels incomplete - update to be:
Yes! Proxy servers for the polling connection that takes place between the agent and Octopus Server are supported - their configuration can be set via the `.pollingProxy.*` helm values.